bus/bus.go

```diff
@@ -835,12 +835,24 @@ func (b *bus) gougingParams(ctx context.Context) (api.GougingParams, error) {
 	}, nil
 }
 
-func (b *bus) accountsOwnerHandlerGET(jc jape.Context) {
-	var owner api.ParamString
-	if jc.DecodeParam("owner", &owner) != nil {
+func (b *bus) accountsHandlerGET(jc jape.Context) {
+	jc.Encode(b.accounts.Accounts())
+}
+
+func (b *bus) accountHandlerGET(jc jape.Context) {
```

Comment:
This endpoint def. feels a bit strange right. You'd expect `GET /account/:id` and not `POST /account/:id` that asks a host key. I considered suggesting `/accounts/host/:hostkey` and keep `/account/:id` for lock and unlock endpoints... 

I think it's fine, I wanted to leave the comment because it might make you reconsider. I do like the current API, only this endpoint feels quirky. Theoretically we should not need two variables if the accounts or shared between workers.

Reply:
It does indeed. I responded with some details on that on your other comment.